spec/xclarity_client_node_spec.rb

```diff
@@ -58,6 +58,19 @@
         end
       end
     end
+
+    context 'attributes included in the version 1.4' do
+      it 'getting new attributes' do
+        uuid = "5D3237D000A711E787D30894EF3C1E99"
+        response = @client.fetch_nodes([uuid])
+        response.each do |node|
+          node.securityDescriptor.should_not be_nil
+          node.primary.should_not be_nil
+          node.logicalID.should_not be_nil
+          node.FeaturesOnDemand.should_not be_nil
```

Comment:
It is important to ensure that this property will be a hash, once the will expect a JSON-like object to parse it.